src/mono/msbuild/android/build/AndroidApp.targets

```diff
@@ -0,0 +1,174 @@
+    </ItemGroup>
+    
+    <MakeDir Directories="$(_MobileIntermediateOutputPath)" />
+  </Target>
+
+  <Target Name="_AndroidAotCompileApp"
+          Condition="'$(RunAOTCompilation)' == 'true'"
+          DependsOnTargets="_AndroidBeforeAotCompileApp">
+
+    <MonoAOTCompiler
+        CompilerBinaryPath="@(MonoAotCrossCompiler->WithMetadataValue('RuntimeIdentifier','$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())'))"
+        OutputDir="$(_MobileIntermediateOutputPath)"
+        Mode="$(_AOTMode)"
+        OutputType="AsmOnly"
+        Assemblies="@(_AotInputAssemblies)"
+        AotModulesTablePath="$(AndroidAppBundleDir)\modules.c"
+        IntermediateOutputPath="$(_MobileIntermediateOutputPath)"
+        MibcProfilePath="$(_MibcProfilePath)"
+        UseLLVM="$(MonoEnableLLVM)"
+        LLVMPath="$(MonoAotCrossDir)">
+        <Output TaskParameter="CompiledAssemblies" ItemName="_AndroidAssembliesInternal" />
+    </MonoAOTCompiler>
+
+    <ItemGroup>
+      <_AndroidAssembliesInternal Include="@(_AOT_InternalForceInterpretAssemblies)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_AndroidPrepareProfiledAot"
+          Condition="'$(RunProfileAOT)' == 'true'"
+          DependsOnTargets="_AndroidBeforeAotCompileApp">
+
+    <NetTraceToMibcConverter
+      MibcConverterBinaryPath="$(DotnetPgoToolPath)"
+      Assemblies="@(_AotInputAssemblies)"
+      NetTracePath="$(NetTracePath)"
+      CacheFilePath="$(IntermediateOutputPath)netrace_to_mibc_converter_cache.json"
+      OutputDir="$(_MobileIntermediateOutputPath)">
+      <Output TaskParameter="MibcProfilePath" PropertyName="_MibcProfilePath" />
+    </NetTraceToMibcConverter>
+
+  </Target>
+
+  <Target Name="_AndroidGenerateAppBundle" DependsOnTargets="_AndroidGenerateRuntimeConfig">
+
+    <AndroidAppBuilderTask
+        RuntimeIdentifier="$(RuntimeIdentifier)"
+        ProjectName="$(AssemblyName)"
+        MonoRuntimeHeaders="$(MicrosoftNetCoreAppRuntimePackNativeDir)include\mono-2.0"
+        Assemblies="@(_AndroidAssembliesInternal)"
+        MainLibraryFileName="$(MainLibraryFileName)"
+        EnvironmentVariables="@(AndroidEnv)"
+        ForceAOT="$(RunAOTCompilation)"
+        ForceInterpreter="$(MonoForceInterpreter)"
+        ProfileAOT="$(RunProfileAOT)"
+        StripDebugSymbols="False"
+        RuntimeComponents="$(RuntimeComponents)"
+        DiagnosticPorts="$(DiagnosticPorts)"
+        OutputDir="$(AndroidAppBundleDir)"
+        AppDir="$(AndroidAppDir)">
+        <Output TaskParameter="ApkPackageId"  PropertyName="ApkPackageId" />
+        <Output TaskParameter="ApkBundlePath" PropertyName="ApkBundlePath" />
+    </AndroidAppBuilderTask>
+
+    <Message Importance="High" Text="PackageId:       $(ApkPackageId)"/>
+    <Message Importance="High" Text="Instrumentation: net.dot.MonoRunner"/>
+    <Message Importance="High" Text="Apk:             $(ApkBundlePath)"/>
+  </Target>
+
+  <Target Name="_AfterAndroidBuildApp">
+    
+  </Target>
```

Comment:
empty, remove?

Reply:
I'd prefer to keep it in in the event we end up needing it.